libraries/AP_Scripting/examples/mission-load.lua

```diff
@@ -1,6 +1,8 @@
 -- Example of loading a mission from the SD card using Scripting
 -- Would be trivial to select a mission based on scripting params or RC switch
 -- luacheck: only 0
+--In case of testing it with SITL on linux - copy this "mission-load.lua" script to "ardupilot/ArduCopter/scripts" directory
```

Comment:
yes, you are correct...
```suggestion
--Copy this "mission-load.lua" script to the "scripts" directory of the simulation or autpilot's SD card. The "mission1.txt" file containing the mission items should be placed in the directory above the "scripts" directory.
```

Reply:
Thanks for considering my request. In case of placing it on sdcard it requires mission item at the same level as APM directory other wise it will not work if it is inside APM directory. This is tested and confirmed with Orange blue flight controller.

Reply:
which is exactly what my suggested wording states

Reply:
Well to be fair @Hwurzburg some people will still find this confusing. 
Would it not make it absolutely clear to add "In case of placing it on SD Card, mission1.txt file should be placed in the APM directory root."

Reply:
@timtuxworth that could be added, but cant replace anything in the suggested text since it applies to SITL, SD,......and ROMFS

Reply:
Yes I'm suggesting to add it to what you proposed.